metadata-ingestion/src/datahub/ingestion/source/openapi_parser.py

```diff
@@ -314,12 +316,10 @@ def extract_fields(
             return ["contains_a_string"], {"contains_a_string": dict_data[0]}
         else:
             raise ValueError("unknown format")
-    if len(dict_data.keys()) > 1:
+    if len(dict_data) > 0:
         # the elements are directly inside the dict
         return flatten2list(dict_data), dict_data
-    dst_key = list(dict_data.keys())[
-        0
-    ]  # the first and unique key is the dataset's name
+    dst_key = list(dict_data)[0]  # the first and unique key is the dataset's name
```

Comment:
don't really understand this change - should the condition still be `> 1` above?

Reply:
I didn't actually mean to include that change, but I made it while I was testing because I don't understand why it was 1 to begin with. In any case, I'll remove it for this PR